docs/examples/amor.ipynb

```diff
@@ -30,15 +38,48 @@
    "outputs": [],
    "source": [
     "pl = sciline.Pipeline(\n",
-    "    (*amor.providers, *amor.data.providers), params=amor.default_parameters\n",
+    "    (\n",
+    "        *amor.providers,\n",
+    "        *amor.data.providers,\n",
+    "    ),\n",
+    "    params=amor.default_parameters,\n",
     ")\n",
     "\n",
-    "pl[QBins] = sc.geomspace(dim='Q', start=0.008, stop=0.075, num=200, unit='1/angstrom')\n",
-    "pl[SampleRotation[Sample]] = sc.scalar(0.7989, unit='deg')\n",
-    "pl[PoochFilename[Sample]] = \"sample.nxs\"\n",
```

Comment:
We should also, if possible, check that the workflow still works on the old files.

Reply:
I tested: it no longer works with the old file (because loading fails as the file structure is different).
Conclusion from discussion: we will not worry about this because the old Amor files were not standard Amor files, they had been tweaked. This Amor module works with files currently being produced at Amor, and should be enough.